packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1419,6 +1426,19 @@ export default class Meeting extends StatelessWebexPlugin {
      */
     this.remoteShareInstanceId = null;
 
+    /**
+     * Status used for ensuring we do not oversend metrics
+     * @instance
+     * @private
+     * @memberof Meeting
+     */
+    this.shareMetricSentStatus = {
```

Comment:
nitpick: since this logic is very specific to call analyzer not metrics in general I would maybe rename it to mention CA specifically, something like `shareCAEventSentStatus`